support/elasticsearch2_search.erl

```diff
@@ -721,6 +722,56 @@ filter_categories(Cats, Context) ->
         Cats
     ).
 
+% Add wildcards to the text unless there are operators.
+add_suffix(Text) ->
+    Text1 = z_convert:to_binary(Text),
+    case        binary:match(Text1, <<"\"">>) =:= nomatch
+        andalso binary:match(Text1, <<"*">>) =:= nomatch
+        andalso binary:match(Text1, <<"-">>) =:= nomatch
+        andalso binary:match(Text1, <<"+">>) =:= nomatch
+        andalso binary:match(Text1, <<"^">>) =:= nomatch
+    of
+        true ->
+            Parts = binary:split(z_convert:to_binary(Text), <<$">>, [ global ]),
```

Comment:
The convert to binary is already done in `Text1`, but if I'm correct this split will never return more than one item, because of the `nomatch` on `"` before

Reply:
I have reworked the wildcard detection.

Also added a couple of options to the textual search.